types/index.d.ts

```diff
@@ -127,7 +139,7 @@ export interface NextAuthOptions {
    * [Default adapter](https://next-auth.js.org/schemas/adapters#typeorm-adapter) |
    * [Community adapters](https://github.com/nextauthjs/adapters)
    */
-  adapter?: Adapter
+  adapter?: Adapter<TUser, TProfile, TSession, TVerificationRequest>
```

Comment:
Won't be needed when you fix the merge conflicts

Reply:
See above for why this is still included.